starter/script.js

```diff
@@ -1 +1,119 @@
-// put code here!
+
+var firstRound = function() {
+  console.log("It's your first day on the job as winter caretaker at the Overlook Hotel.  You and your family are looking forward to a nice relaxing winter.");
+  buffer();
+  console.log("So, What do you want to do first?\nWork on your novel?\nGo see what Danny is up to?\nOr ride your big wheel around the hotel?");
+  buffer();
+  console.log("Please enter '1' to work on your novel, '2' to go see Danny, or '3' to ride the big wheel.")
+  buffer();
+  var firstInput = prompt("Please enter either \n'1': to work on your novel,\n'2': to go see Danny,\n'3': to ride the big wheel.");
+
+  if (firstInput == 1) {
+    console.log("Ok, you chose to work on your novel.");
+    console.log("How’s this for an opening line? “All work and no play make Jack a dull boy”");
+    buffer();
+    secondRound();
+  }
+  else if (firstInput == 2) {
+    console.log("Ok, you decided to go see Danny");
+    console.log("Danny was busy talking to his friend Tony about red wine, or something like that.");
+    buffer();
+    secondRound();
+  }
+  else if (firstInput == 3) {
+    console.log("Ok, you decided to go for a ride on the big wheel.");
+    console.log("You ran into some adorable little twin girls.  They were super friendly and wanted you to come play with them forever and ever and ever...");
+    buffer();
+    secondRound();
+  }
+}
+buffer();
+
```

Comment:
really like the modular approach you took here by breaking up your code into reusable functions!